CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj git fetch` now accepts `-b` as a shorthand for `--branch`, making it more
   consistent with other commands that accept a branch
 
+* `jj config list` gained a `--show-sources` flag to display the origin of
```

Comment:
Thanks, this looks like a useful feature. The equivalent option in Git is called `--show-origin`, and here and in other places, you also describe the information displayed as an "origin", so it might be better to use that term. (You can hold off making that change until other people have chimed in on the name.)

Reply:
I should have referenced https://github.com/martinvonz/jj/issues/1047 in the PR, will add that now. That has a task to add `--show-sources`, but I'm happy to switch to `--show-origin` if preferred.

Reply:
Updated flag name to `--show-origin`.